SpectraLogic.EscapePodClient/EscapePodClient.cs

```diff
@@ -0,0 +1,48 @@
+﻿using log4net;
+using SpectraLogic.EscapePodClient.Calls;
+using SpectraLogic.EscapePodClient.Model;
+using SpectraLogic.EscapePodClient.ResponseParsers;
+using SpectraLogic.EscapePodClient.Runtime;
+
+namespace SpectraLogic.EscapePodClient
+{
+    public class EscapePodClient : IEscapePodClient
+    {
+        private readonly ILog _log = LogManager.GetLogger("EscapePodClient");
+        private readonly INetwork _network;
+
+        internal EscapePodClient(INetwork network)
+        {
+            _network = network;
+        }
+
+        public void VerifyArchiveName(string archiveName)
```

Comment:
What does Verify Archive Name do exactly?

Reply:
its a placeholder for verify that the archive exists